### ISSUE TITLE
inferences & bootstrap

### ISSUE THREAD

**Issue by vincentarelbundock** [vincentarelbundock/pymarginaleffects#67]

**Comment by artiom-matvei** [comment]
`inferences()` function present in R but not in python.

is this the issue to add this function?

@vincentarelbundock

**Comment by vincentarelbundock** [comment]
Yes, but this is a big project that can wait. I want to have full documentation before working on more features. Also, this one will be a lot of work and is more of a priority: https://github.com/vincentarelbundock/pymarginaleffects/issues/115

**Comment by artiom-matvei** [comment]
@vincentarelbundock since #115 is closed, can I tackle this feature, i.e. adding `inferences()` function?

**Comment by vincentarelbundock** [comment]
I think this is going to require some architecture, and is likely to be a bit challenging. 

The main issue is that some of the methods require fitting the model again, but the output of `marginaleffects` functions does not include all the info needed for that.

I would prefer you focus on the other issues I tagged for you.

When I find some time, I'll try to think of a design, and then we can talk about it. Unfortunately the next couple weeks are probably too busy for me to look at this.
